terraform/scripts/bamboo/deploy-database.sh

```
@@ -0,0 +1,91 @@
+# | bamboo_AWS_SECRET_ACCESS_KEY    | Yes      | -                              |
+# | bamboo_AWS_DEFAULT_REGION       | No       | us-east-1                      |
+# | bamboo_DB_INSTANCE_CLASS        | No       | db.t3.medium                   |
+# | bamboo_ALLOCATED_STORAGE        | No       | 20                             |
+# | bamboo_MAX_ALLOCATED_STORAGE    | No       | 100                            |
+# | bamboo_DB_ENGINE_VERSION        | No       | 17.4                           |
+# | bamboo_DELETION_PROTECTION      | No       | true                           |
+# +---------------------------------+----------+--------------------------------+
+
+# Set AWS credentials from Bamboo variables
+export AWS_DEFAULT_REGION="${bamboo_AWS_DEFAULT_REGION:-us-east-1}"
+export AWS_ACCESS_KEY_ID="${bamboo_AWS_ACCESS_KEY_ID}"
```

Shellcheck (0.11.0)

[warning] bamboo_AWS_ACCESS_KEY_ID is referenced but not assigned.

(SC2154)